go/vt/mysqlctl/backup.go

```diff
@@ -103,7 +103,7 @@ func init() {
 }
 
 func registerBackupFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&backupStorageHook, "backup_storage_hook", backupStorageHook, "if set, we send the contents of the backup files through this hook.")
+	fs.StringVar(&backupStorageHook, "backup_storage_hook", backupStorageHook, "if set, we send the contents of the backup files through this hook (deprecated: hooks will be disabled in v16.0. Please use ExternalCompressorCmd/ExternalDecompressorCmd instead).")
```

Comment:
instead of doing this, using [MarkDeprecated](https://pkg.go.dev/github.com/spf13/pflag#readme-deprecating-a-flag-or-its-shorthand), which will also hide it from the help output

Reply:
fixed